docs/api/console.md

```diff
@@ -0,0 +1,123 @@
+# 🎮 console
+
+An implementation of the standard [`console`](https://developer.mozilla.org/en-US/docs/Web/API/console) API.
+
+## Quick start
+
+The console API provides access to the `jztz` debug log. 
+General purpose mesages can be created using [`console.log()`](#log)
+```typescript
+console.log("Hello from JS 👋");
+```
+For messages with a more specific purpose the following functions are provided:
+[`console.info`](#info), [`console.warn`](#warn), [`console.error`](#error), [`console.debug`](#debug).
+
+These functions can be combined to make logs more informative.
+```typescript
+function myAwesomeFunction(arg: unknown) {
+    console.debug(`Entered myAwesomeFunction with argument ${arg}`)
+    if (typeof arg !== 'string') {
+    console.warn("Converting non string argument to string");
+    arg = arg.toString();
+    }
+    try {
+        let result = functionThatMayThrow(arg);
+        console.info(`Result: ${result}`);
+    } catch (error) {
+        console.error(`Call failed with message ${error}`);
+    }
+}
+```
+## Logging objects 
+`console` functions are able to output a string representation for standard javascript objects.
+All logging functions can be called with multiple arguments, the message will consist of the string representations of each of the arguments
+```
+const dave = { name: "Dave", age: 42 };
+console.log("Hello", dave);
```

Comment:
Our console API simply doesn't support this. This section should explicitly mention the use of `JSON.stringify` to log objects.

Reply:
I think Leo is fixing this.

Reply:
While it is in our backlog, no one is actively working on this. @ryutamago's work pertains to the logging _service_, not the inspection of `JsObject`s in the logs. Given this the docs should mention this and explicitly point to `JSON.stringify` for printing simple objects